src/Http/Middleware/GripMiddleware.php

```diff
@@ -114,7 +114,7 @@ function setup_grip(Request $request ) {
                     /** @var JwtAuth $auth */
                     $auth = $client->auth;
                     Log::debug('validating: ' . $grip_sig . ' with ' . $auth->key );
```

Comment:
```suggestion
                    Log::debug('validating: ' . $grip_sig . ' with ' . $client->get_verify_key() );
```
I'd imagine you'd need this in case $auth is a BearerAuth